compact_test.go

```diff
@@ -747,34 +748,70 @@ func TestDisableAutoCompactions(t *testing.T) {
 // TestCancelCompactions ensures that when the db is closed
 // any running compaction is cancelled to unblock closing the db.
 func TestCancelCompactions(t *testing.T) {
-	tmpdir, err := ioutil.TempDir("", "test")
-	testutil.Ok(t, err)
-	defer os.RemoveAll(tmpdir)
+	createTestDb := func() (*DB, func()) {
+		tmpdir, err := ioutil.TempDir("", "testCancelCompaction")
+		testutil.Ok(t, err)
 
-	// Create some blocks to fall within the compaction range.
-	createBlock(t, tmpdir, 3000, 0, 1000)
-	createBlock(t, tmpdir, 3000, 1000, 2000)
-	createBlock(t, tmpdir, 1, 2000, 2001) // The most recent block is ignored so can be e small one.
+		// Create some blocks to fall within the compaction range.
+		createBlock(t, tmpdir, 4000, 0, 1000)
```

Comment:
Just curious why change in num of series `3000` -> `4000` ?

Reply:
gives bigger difference between normal run vs cancelled so it is less likely to false fail.